core/modules/user/user.admin.inc

```diff
@@ -326,7 +326,11 @@ function user_login_settings($form, &$form_state) {
   );
   $form['ip_limit_settings']['help'] = array(
     '#type' => 'item',
-    '#description' => t('Do not allow any login from the current user\'s IP if the limit has been reached. This is independent of the per-user limit to catch attempts from one IP to log in to many different user accounts. By default we have a reasonably high limit since there may be only one apparent IP for all users at an institution.'),
+    '#description' => t("Do not allow any further login attempts <em>from the current user's IP</em>, if the limit has been reached. This is used to prevent malicious login attempts from one IP using many different user accounts. By default the limit is set reasonably high, to account for legitimate use cases (for example, multiple users from the same institution logging in from a common IP)."),
```

Comment:
```suggestion
    '#description' => t("Do not allow any further login attempts <em>from the current user's IP</em> if the limit has been reached. This setting is used to prevent malicious login attempts from one IP using many different user accounts. By default the limit is set relatively high to account for legitimate use cases, for example, multiple users from the same institution logging in from a common IP."),
```
Some grammatical/punctuation suggestions.